sysid-application/src/integrationtest-generation/native/cpp/GenerationTest.cpp

```diff
@@ -145,13 +135,13 @@ TEST_F(GenerationTest, Drivetrain) {
   // Encoders + Motor controllers already tested, now just need to test the
   // gyros
   for (auto&& gyro : sysid::kGyros) {
-    if (wpi::StringRef(gyro) == "Romi") {
+    if (std::string_view{gyro} == "Romi") {
       continue;
     }
     m_settings.m_gyro = gyro;
     for (auto&& gyroCtor : gyroCtorMap[gyro]) {
       m_settings.m_gyroCtor = gyroCtor;
-      wpi::outs() << "Testing: " << gyro << " using " << gyroCtor << "\n";
+      fmt::print("Testing: {} using {}\n", gyro, gyroCtor);
```

Comment:
It also seems like this print isn't being printed out in order: https://github.com/wpilibsuite/sysid/pull/144/checks?check_run_id=2702756388. These should precede the output from deploying the simulation, but it only gets printed at the end.